README.md

```diff
@@ -188,4 +188,4 @@ This shows that what you've run actually started, accessed the internal modules
 
 ## How to contribute to the project?
 
-Please check [the how to contribute](https://github.com/facundobatista/pyempaq/CONTRIBUTING.md) instructions.
+Please check [the how to contribute](CONTRIBUTING.md) instructions.
```

Comment:
Pero esto no funcionaría si hacés click cuando lo estás leyendo en Read The Docs, no?

Reply:
Tenés razón. Lo probé solamente desde Github editando el Readme.

Te parece que quede igual que los otros que funcionan? => _https://github.com/facundobatista/pyempaq/blob/main/CONTRIBUTING.md_